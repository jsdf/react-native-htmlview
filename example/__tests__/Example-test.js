import React from 'react';

import renderer from 'react-test-renderer';

import Example from '../Example';

describe('<Example/>', () => {
  it('should render', () => {
    expect(renderer.create(<Example />).toJSON()).toMatchSnapshot();
  });
});
