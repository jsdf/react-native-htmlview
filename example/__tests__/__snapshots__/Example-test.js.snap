// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Example/> should render 1`] = `null`;
